tests/integration/informer_mock_test.go

```diff
@@ -0,0 +1,29 @@
+package biz
+
+import (
+	"context"
+	"github.com/docker/docker/api/types"
+)
+
+const (
+	containerID = "my-container"
+	pid         = 666
+)
```

Comment:
Nit: as those constants are defined for the whole `tests/integration` package I would be a bit more verbose with the naming.
```go
const (
	inspectorDefaultContainerID = "my-container"
	inspectorDefaultPid                = 666
)
```

Reply:
done